feedback-module/src/lib/hooks/useForm.js

```diff
@@ -10,47 +10,35 @@ import { flattenInputs } from "../utils/textboxUtil";
 export default function useForm() {
   const [formData, setFormData] = useState({});
 
-  const setFeedbackType = (text, feedbackID) => {
-    let updatedData = formData;
-    updatedData.feedbackType = {
-      label: text,
-      feedbackID: feedbackID,
-    };
-    setFormData(updatedData);
-  };
-
-  const setCheckedOptions = (checkedFields) => {
-    let updatedData = formData;
-    if (checkedFields) {
-      updatedData.checkedOptions = flattenCheckboxes(checkedFields);
-    } else {
-      updatedData.checkedOptions = [];
-    }
-    setFormData(updatedData);
-  };
-
-  const setInputResponses = (inputQuestions) => {
-    let updatedData = formData;
-    if (inputQuestions) {
-      updatedData.inputResponses = flattenInputs(inputQuestions);
-    } else {
-      updatedData.inputResponses = [];
-    }
-    setFormData(updatedData);
-  };
-
-  const setSource = () => {
+  const setSubmission = (
+    type,
+    text,
+    feedbackID,
+    checkedFields,
+    inputQuestions
+  ) => {
     let updatedData = formData;
```

Comment:
Maybe add a few comments explaining when you're setting checked options, input options, etc... Like still kind of separate it as if they were separate functions just so it is more clear for a code reader